ui/src/router/details/logs/RouterLogsView.js

```diff
@@ -55,24 +67,73 @@ export const RouterLogsView = ({ router }) => {
     (entries) =>
       !!entries.length ? entries[entries.length - 1].timestamp : undefined,
     (entries) => entries.map((entry) => LogEntry.fromJson(entry).toString()),
-    configOptions
+      appConfig.podLogs,
   );
 
   const availableComponents = components.filter(
     (c) => c.value === "router" || !!get(router, `config.${c.value}`)
   );
 
+  useEffect(
+    () => {
+      let urls = {}
+      if (appConfig.imagebuilder.cluster && appConfig.imagebuilder.gcp_project && appConfig.imagebuilder.namespace &&
```

Comment:
does this need to be here for can `createStackdriverUrl` import it?

Reply:
It has to be there since Turing can only access the app configs via a config provider https://github.com/caraml-dev/turing/blob/91e22116cad1eb3b70ec100be94d683c8fa0b092/ui/src/config.js#L115 unlike Merlin which exposes the app configs as an exported variable: https://github.com/caraml-dev/merlin/blob/8edef22b29d0bfb2728d62b1f880f1f753f9509e/ui/src/config.js#L29. 

I can make the app configs exported also, but to be honest I'm not sure why we're using a provider if we're already exposing it globally 😅

Reply:
Okay I've decided to just export the app configs, just like how it's done in Merlin. This keeps the code consistent across both codebases.